troubleshooting/known_issues/Gigabye_Missing_Redfish_Data.md

```diff
@@ -0,0 +1,24 @@
+### Gigabyte BMC Missing Redfish Data
+
+Follow this procedure if you notice data from Gigabyte nodes are missing from Hardware State Manager(HSM) or other CSM tools.
+Make sure the firmware and BIOS of the Gigabyte node is at the latest supported level.
+
+If data from Gigabyte nodes are missing from Hardware State Manager (HSM) or other CSM tools, check the redfish endpoint on the BMC to see if the data is present.

```

Comment:
- No need to spell out Hardware State Manager again. You only need to use HSM.
- Capitalize Redfish to be consistent.

Reply:
Updated